pkgs/development/python-modules/dnslib/default.nix

```
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "dnslib";
+  version = "0.9.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0w8spp4fnw63xx9zj77zvgk1qabd97ifrj6gap2j36gydnarr42c";
+  };
+
+  checkPhase = "VERSIONS=python ./run_tests.sh";
```

```suggestion
  checkPhase = "VERSIONS=${python.interpreter} ./run_tests.sh";
```
Because e.g. the PyPy executable isn't called python.